pkgs/development/python-modules/libevdev/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "libevdev";
+  version = "0.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "10gwj08kn2rs4waq7807mq34cbavgkpg8fpir8mvnba601b8q4r4";
+  };
+
+  doCheck = false;
```

Comment:
Why were the tests disabled?

Reply:
They crash because they can't find libraries.

Reply:
Can you add a small comment with an example? Maybe s.o can find time to debug it.
It's preferrable to run the tests if possible, so breakages caused by other python package bumps are more likely to be noticed.

Reply:
crashing isn't a good sign, could mean that it's broken in a sandbox, thus nixpkgs